src/app.ts

```diff
@@ -20,12 +21,13 @@ const main = async () => {
 
   await dbConnection();
 
-  var bodyParser = require('body-parser');
+  // eslint-disable-next-line vars-on-top, global-require, @typescript-eslint/no-var-requires
+  const bodyParser = require('body-parser');
 
-  var app = Express();
+  const app = Express();
 
-  app.use(bodyParser.urlencoded({ extended: false }))
-  app.use(bodyParser.json())
+  app.use(bodyParser.urlencoded({ extended: false }));
+  app.use(bodyParser.json());
```

Comment:
should probably have the linter detect no semicolons too if that's possible

Reply:
added